### PR TITLE
Rename colors

### PR DESCRIPTION
**Pull request by marciovicente**
This pull request rename colors: variables and classes, using Monalisa pattern. 

Basically:
```css
$colorName-A100 => $colorName-12
$colorName-A200 => $colorName-26
$colorName-A400 => $colorName-54
$colorName-A700 => $colorName-87


$( black | white )-1 => $( black | white )-87
$( black | white )-2 => $( black | white )-54
$( black | white )-3 => $( black | white )-26
$( black | white )-4 => $( black | white )-12
```

### CODE REVIEW

**Comment by matheusrocha89** [comment]
LGTM

**Comment by andersonba** [comment]
LGTM

**Comment by rafaelverger** [comment]
LGTM

File: core/scss/base/_colors.scss [hunk 1]
```diff
@@ -250,44 +250,44 @@
...
-.bgc-orange-A400             {   background-color: $orange-A400 !important;       }
-
-// A700
-.bgc-red-A700                {   background-color: $red-A700 !important;          }
-.bgc-blue-A700               {   background-color: $blue-A700 !important;         }
-.bgc-green-A700              {   background-color: $green-A700 !important;        }
-.bgc-yellow-A700             {   background-color: $yellow-A700 !important;       }
-.bgc-orange-A700             {   background-color: $orange-A700 !important;       }
+// 12
+.bgc-red-12                  {   background-color: $red-12 !important;            }
```
Comment (tupy) [inline core/scss/base/_colors.scss]: `bg-red12`

File: core/scss/base/_colors.scss [hunk 2]
```diff
@@ -103,47 +103,47 @@
...
-.tc-orange-A400             {   color: $orange-A400 !important;                  }
-
-// A700
-.tc-red-A700                {   color: $red-A700 !important;                     }
-.tc-blue-A700               {   color: $blue-A700 !important;                    }
-.tc-green-A700              {   color: $green-A700 !important;                   }
-.tc-yellow-A700             {   color: $yellow-A700 !important;                  }
-.tc-orange-A700             {   color: $orange-A700 !important;                  }
+// 12
+.tc-red-12                  {   color: $red-12 !important;                       }
```
Comment (tupy) [inline core/scss/base/_colors.scss]: `red12`

**Pull request merged**